cmd/server/assets/users/import.html

```diff
@@ -67,6 +67,17 @@ <h1>Import users</h1>
         </table>
       </div>
     </div>
+
+    <div class="card mb-3 shadow-sm">
+      <div class="card-header">Import</div>
+      <div class="card-body">
+        <p>
+          Use the following link to export a list of users.
+        </p>
+        <a class="card=link" href="/realm/users/export.csv">export.csv</a>
```

Comment:
`target="_blank"` ??

Reply:
Done. That said the content-disposition header should cause the browser to download rather than try to display this content.